scene/gui/dialogs.cpp

```diff
@@ -363,8 +362,7 @@ Button *ConfirmationDialog::get_cancel_button() {
 
 ConfirmationDialog::ConfirmationDialog() {
 	set_title(TTRC("Please Confirm..."));
-#ifdef TOOLS_ENABLED
-	set_min_size(Size2(200, 70) * EDSCALE);
-#endif
+	set_min_size(Size2(200, 70));
```

Comment:
I just plain removed the scale here. I get that we lose scaling for the default size of the dialog, but I don't know what a proper solution can be here. Classes that extend this dialog or reuse this dialog already lose that scaling and need to carry for it themselves. In user projects running with HiDPI this is also a problem.

And we can't read the scale here because there is no theme yet, as we are not in the tree yet. Maybe a scaling factor should be a part of the base Window class and be applied automatically on top of the min size?

Reply:
For the record, this also means that ConfirmationDialog in projects will now have this min size of `200x70`. It might be a weirdly specific magic number but well, it's better than zero.

> And we can't read the scale here because there is no theme yet, as we are not in the tree yet. Maybe a scaling factor should be a part of the base Window class and be applied automatically on top of the min size?

That might make sense yeah, and solve many issues where this had to be hardcoded in editor code. Might be for a follow-up PR maybe to keep the scope manageable here.